README.md

```diff
@@ -114,10 +108,6 @@ func HandleLambdaRequest() {
 }
 
 func main() {
-	//Wrapping with wflambda.Wrapper
 	lambda.Start(wflambda.Wrapper(HandleLambdaRequest))
```

Comment:
why remove comments?  
wflambda.Wrapper(handler)

Reply:
I wanted the comments in this example to focus on the addition of the custom metrics (the comments I added in the earlier example focus on why the wrapper is used). I've also corrected the function call.